### ISSUE TITLE
Implement File Manager DBus Interface (org.freedesktop.FileManager1)

### ISSUE THREAD

**Issue by CyanChanges** [sxyazi/yazi#1698]
### `yazi --debug` output

```Shell
Yazi
    Version: 0.3.3 (3e4973db 2024-09-27)
    Debug  : false
    OS     : linux-x86_64 (unix)

Ya
    Version: 0.3.3 (3e4973db 2024-09-27)

Emulator
    Emulator.via_env: ("xterm-kitty", "")
    Emulator.via_csi: Ok(Kitty)
    Emulator.detect : Kitty

Adapter
    Adapter.matches: Kitty

Desktop
    XDG_SESSION_TYPE           : Some("wayland")
    WAYLAND_DISPLAY            : Some("wayland-1")
    DISPLAY                    : Some(":0")
    SWAYSOCK                   : None
    HYPRLAND_INSTANCE_SIGNATURE: Some("58669fef77ac17ea205ce3570f48e17de736111f_1727533958_140585338")
    WAYFIRE_SOCKET             : None

SSH
    shared.in_ssh_connection: false

WSL
    WSL: false

Variables
    SHELL              : Some("/usr/bin/zsh")
    EDITOR             : Some("nvim")
    VISUAL             : None
    YAZI_FILE_ONE      : None
    YAZI_CONFIG_HOME   : None

Text Opener
    default: Some(Opener { run: "${EDITOR:=vi} \"$@\"", block: true, orphan: false, desc: "$EDITOR", for_: None, spread: true })
    block  : Some(Opener { run: "${EDITOR:=vi} \"$@\"", block: true, orphan: false, desc: "$EDITOR", for_: None, spread: true })

Multiplexers
    TMUX               : false
    tmux version       : 3.5
    ZELLIJ_SESSION_NAME: None
    Zellij version     : 0.40.1

Dependencies
    file             : 5.45
    ueberzugpp       : No such file or directory (os error 2)
    ffmpegthumbnailer: 2.2.3
    magick           : 7.1.1-38
    fzf              : 0.55.0
    fd               : 10.2.0
    rg               : 14.1.1
    chafa            : 1.14.2
    zoxide           : 0.9.6
    7z               : 24.08
    7zz              : No such file or directory (os error 2)
    jq               : 1.7.1


--------------------------------------------------
When reporting a bug, please also upload the `yazi.log` log file - only upload the most recent content by time.
You can find it in the "/home/cyan/.local/state/yazi" directory.
```

### Please describe the problem you're trying to solve

I am trying to use yazi to replace my GUI File Manager.

I tried to make open downloaded files with yazi in Chrome...
but after messing with many things like the mimeapps, desktop files, gtk-launch
I only get yazi launched in the `Downloads/` folder, which not selecting on the target file
![image](https://github.com/user-attachments/assets/b93271fb-b8cb-4ae4-b80f-5e7b67a7e067)

I notice that Chrome use the `.ShowItems` of `org.freedesktop.FileManager1` when click `Show in the folder`
![image](https://github.com/user-attachments/assets/1bc272c4-1a74-4043-846f-2597bb5f9c80)
and after it fails, it calls the default in mime types (which is yazi), and only pass the `Downloads/` directory, 
not the targeting file.

TL;DR many apps like chrome use `org.freedesktop.FileManager1` to open a file in the FM, but Yazi didn't implement it, so it didn't open correctly. 

### Would you be willing to contribute this feature?

- [ ] Yes, I'll give it a shot

### Describe the solution you'd like

If Yazi implements the `org.freedesktop.FileManager1` DBus protocol,

many apps like Chrome could open files in Yazi easily.
and I can use Yazi to replace all GUI file manager.

### Additional context

![image](https://github.com/user-attachments/assets/b93271fb-b8cb-4ae4-b80f-5e7b67a7e067)
<small> Yazi launched in `Downloads/` and not selecting the targeting files </small>

![image](https://github.com/user-attachments/assets/1bc272c4-1a74-4043-846f-2597bb5f9c80)
<small> Chrome calls `org.freedesktop.FileManager1.ShowItems`, and failed </small> 

### Validations

- [X] I have searched the existing issues/discussions
- [X] The latest nightly build of Yazi doesn't already have this feature

**Comment by sxyazi** [comment]
Duplicate of https://github.com/sxyazi/yazi/discussions/1119

https://github.com/boydaihungst/xdg-desktop-portal-termfilechooser is exactly what you're looking for

**Comment by CyanChanges** [comment]
> Duplicate of #1119
> 
> https://github.com/boydaihungst/xdg-desktop-portal-termfilechooser is exactly what you're looking for

I just tried that, but it didn't work, everything works same.
maybe i configured something wrong but it didn't get much document.

**Comment by CyanChanges** [comment]
> > Duplicate of #1119
> > https://github.com/boydaihungst/xdg-desktop-portal-termfilechooser is exactly what you're looking for
> 
> I just tried that, but it didn't work, everything works same. maybe i configured something wrong but it didn't get much document. May be I'll figured out tomorrow.

If it's still I'll respond tomorrow. Thanks tor your responding so quickly.
